stage #1/module #1. Intro, Infrastructure & Instruments, Fundamentals/lesson #3. Classes, Intro to OOP/Weapon/src/test/java/com/epam/training/weapons/logic/DivisionLogicTest.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.training.weapons.logic;
+
+import com.epam.training.weapons.model.AbstractWeapon;
+import com.epam.training.weapons.model.Division;
+import com.epam.training.weapons.model.Gun;
+import com.epam.training.weapons.model.MachineGun;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class DivisionLogicTest {
+    @Test
+    public void calculateTotalWeightTest(){
```

Comment:
split this test into 3, remember 'given', 'when', 'then' structure, comment in your test these parts. 'when' should have only one line. Name tests like testCalculateTotalWeightShouldReturnZeroWhenDevisionIsEmpty

Reply:
Done